src/WormholeJoin.sol

```diff
@@ -135,33 +137,32 @@ contract WormholeJoin {
     * @dev Internal function that executes the withdrawl after a wormhole is registered
```

Comment:
Typo should be `withdrawal` not `withdrawl`. This mistake is made 3 times this file. Also, should be updated to `mint` anyways.

Reply:
I could feel there was something wrong in that word but my laziness just left me inventing a new English word :P
Will change it!

Reply:
done!